components/campaigns/AnnouncementRamainingDays.js

```diff
@@ -17,7 +17,7 @@ export default function AnnouncementRemainingDays({ dateTime, eventName }) {
   }
 
   return (
-    <span className="font-extrabold countdown-text-gradient block">
+    <span className="font-extrabold countdown-text-gradient block" data-testid="span-text">
```

Comment:
Oh my god. Don't use so generic names @reachaadrika 

Use the component name, helpful for code readibility purpose

Reply:
resolved done